spec/awesome_spawn_spec.rb

```diff
@@ -141,6 +141,20 @@
       it "contains #error" do
         expect(subject.send(run_method, "echo 'bad' >&2 && false").error).to eq("bad\n")
       end
+
+      it "combines output when using :capture2e" do
+        result = subject.send(run_method, "echo 'bad' >&2 && false", :capture2e => true)
```

Comment:
I think the spec might be more robust with something like `echo 'good' && echo 'bad' >&2` so there something on each pipe.

Reply:
Sure.  I was just copy-pastaing from above.  I can change.